tardis/tardis_portal/templates/tardis_portal/ajax/access_list_user.html

```diff
@@ -12,15 +12,14 @@ <h3>Add new user</h3>
   <br/>
   {% for field in addUserPermissionsForm %}
 	  {% if field.label == "Authentication Method" %}
```

Comment:
Happy to merge this PR as is, but the removal of the Authentication Method select box in the form without the removal of the 'if field.label == "Authentication Method"' logic is a bit ugly.  It points to a deeper issue (which can't be adequately addressed in this PR) - that there is no longer a good reason to use Django Forms here, especially if we move towards a component based UI, where the UI logic is done on the Javascript side with the server application only providing JSON data via the API.